content/en/profiler/enabling/supported_versions.md

```diff
@@ -9,46 +9,46 @@ further_reading:
+- **Feature-complete versions** give you access to **all** supported features. It's usually best if you update to the latest version of all tracers.
+
+<div class="alert alert-info">For more details, click the language heading in any table to go that language's setup page.</div>
 
-For more details, go to a language's profiler setup page (click the language heading in each table to access).
 
 ## Runtime and tracer versions
 
-To use the Datadog Profiler, use at least the minimum versions summarized in the following table:
+To use the Datadog Profiler, use at least the minimum versions summarized in the following table. To use specific profile types, see [Profile types](#profile-types).
 
 |                                   |  [Java][1]   |   [Python][2]    |    [Go][3]    |   [Ruby][4]    | [Node.js][5]  |  [.NET][6]  |   [PHP][7]    | [Rust/C/C++][8] |
 |-----------------------------------|:------------:|:----------------:|:-------------:|:--------------:|:-------------:|:-----------------------------------------------------------------------:|:-------------:|:---------------:|
 | <strong>Minimum&nbsp;runtime&nbsp;version</strong> | [JDK&nbsp;8+][17]  | Python&nbsp;2.7+ | Go&nbsp;1.19+ | Ruby&nbsp;2.3+ | Node&nbsp;14+ | .NET&nbsp;Core&nbsp;2.1+, .NET&nbsp;5+, .NET&nbsp;Framework&nbsp;4.6.1+ | PHP&nbsp;7.1+ |                 |
-| <strong>Recommended runtime version</strong>       | [JDK&nbsp;11+][17] | Python&nbsp;3.6+ | Go&nbsp;1.21+ | Ruby&nbsp;3.1+ | Node&nbsp;18+ |                              .NET&nbsp;6+                               | PHP&nbsp;8.0+ |                 |
-| <strong>Minimum tracer version</strong>            |              |      0.35.0      |    1.23.0     |     0.48.0     |    0.23.0     |                                  2.7.0                                  | 0.71.0        |                 |
-| <strong>Recommended tracer version</strong>        | [latest][9]  |   [latest][10]   | [latest][11]  |  [latest][12]  | [latest][13]  |                              [latest][14]                               | [latest][15]  |  [latest][16]   |
+| <strong>Feature-complete runtime version</strong>       | [JDK&nbsp;11+][17] | Python&nbsp;3.6+ | Go&nbsp;1.21+ | Ruby&nbsp;3.1+ | Node&nbsp;18+ |                              .NET&nbsp;7+                               | PHP&nbsp;8.0+ |                 |
+| <strong>Feature-complete tracer version</strong>        | [latest][9]  |   [latest][10]   | [latest][11]  |  [latest][12]  | [latest][13]  |                              [latest][14]                               | [latest][15]  |  [latest][16]   |
 
 ## Profile types
 
-To collect profile types for each language, use the minimum versions summarized in the following table:
+To collect profile types, use at least the minimum versions summarized in the following table. If a runtime isn't specified, the profile type requires the minimum runtime version in [Runtime and tracer versions](#runtime-and-tracer-versions).
```

Comment:
> use at least the minimum versions summarized

I'm... still concerned and don't think we should do this :(

We're stating that customers can use a 3-year-old Ruby library version and a 4-year-old Python version. And technically... they can. Probably. Nobody's really checking that such versions still work. And even if they do still "work", they'll provide an incredibly sub-par experience.

Do we really want to tell such customers in 2024+ to go ahead and use these versions?

Reply:
Thanks for the feedback! I think it's less telling users to go ahead and use these versions, and it's more so the bare minimum that a user would need in order to use at least some profiling features.

This is for a user coming in and wanting to see at a glance if they are able to use profiling.

@chrisnas,  @gturbat , and I had a discussion this week, and after some back and forth, we agreed that there is value including this info. I'll let them chime in if they have more to say from their perspective too.